immuni_exposure_ingestion/helpers/risk_level.py

```diff
@@ -64,3 +63,17 @@ def extract_keys_with_risk_level_from_upload(upload: Upload) -> Iterable[Tempora
     )
 
     return keys_at_risk_filtered
+
+
+def set_highest_risk_level_from_upload(
```

Comment:
This method can be protected, i.e., `_` prefix.

Reply:
it's used also in process_upload_eu class!

Reply:
I'd just change the name to `set_highest_risk_level`. Eventually, this method is `upload` agnostic, thus `_from_upload` could be dropped, IMHO.